x-pack/test/functional/apps/spaces/spaces_selection.ts

```diff
@@ -21,14 +21,83 @@ export default function spaceSelectorFunctionalTests({
     'security',
     'spaceSelector',
   ]);
+  const spacesService = getService('spaces');
 
   describe('Spaces', function () {
     before(async () => {
-      await esArchiver.load('x-pack/test/functional/es_archives/spaces/selector');
+      // await esArchiver.load('x-pack/test/functional/es_archives/spaces/selector');
+      await spacesService.create({
```

Comment:
__suggestion:__ how about just this (requires changing `search for "ce 1" and find one space` test to `search for "ce-1 name" and find one space` to account for a new name)?
```ts
const testSpacesIds = ['another-space', ...Array.from('123456789', (idx) => `space-${idx}`)];
before(async () => {
  for (const testSpaceId of testSpacesIds) {
    await spacesService.create({ id: testSpaceId, name: `${testSpaceId} name` });
  }
});
after(async () => {
  for (const testSpaceId of testSpacesIds) {
    await spacesService.delete(testSpaceId);
  }
});
```

Reply:
Data driven for the win :)

Reply:
great suggestion

Reply:
> requires changing search for "ce 1" and find one space test to search for "ce-1 name" and find one space to account for a new name)?

Don't forget this part :slightly_smiling_face: